openstack/compute/v2/extensions/floatingip/requests.go

```diff
@@ -79,7 +112,33 @@ func Associate(client *gophercloud.ServiceClient, serverId, fip string) Associat
 	return res
 }
 
+// AssociateFloatingIP pairs an allocated floating IP with an instance.
+func AssociateFloatingIP(client *gophercloud.ServiceClient, opts AssociateOpts) AssociateResult {
```

Comment:
Would it make sense to name this `AssociateInstance` so that when it called it would be `floatingip.AssociateInstance` instead of `floatingip.AssociateFloatingIP`? Similar proposal for `DisassociateFloatingIP`.